tasks/release_tests.py

```diff
@@ -140,5 +141,62 @@ def test_absent_patch_less_than_any(self):
         )
 
 
+class TestIsDictVersionField(unittest.TestCase):
+    def test_non_version_fields_return_false(self):
+        for key in ['FOO', 'BAR', 'WINDOWS_DDNPM_DRIVER', 'JMXFETCH_HASH']:
+            self.assertFalse(release._is_dict_version_field(key))
+
+    def test_version_fields_return_true(self):
+        for key in ['FOO_VERSION', 'BAR_VERSION', 'JMXFETCH_VERSION']:
+            self.assertTrue(release._is_dict_version_field(key))
+
+    def test_version_fields_return_false_for_win_ddnpm_version(self):
+        self.assertFalse(release._is_dict_version_field('WINDOWS_DDNPM_VERSION'))
```

Comment:
Same test as https://github.com/DataDog/datadog-agent/pull/8021/files#diff-13d397af774de4db215fac599e01c2b6870a25f8840329ed7d4e65838c91774bR146.

Reply:
I see how they seem similar but they're covering different use cases:
- L146 tests all non-`_VERSION` suffixes to eval to `False`
- L149 tests all`_VERSION` suffixes to eval to `True`
- L153 tests that a unlike other examples, a specific field with a `_VERSION` suffix (`WINDOWS_DDNPM_VERSION `) is treated differently and evaluates to `False` just like other non-`_VERSION` suffix fields.

Reply:
I read too quickly and believed `WINDOWS_DDNPM_DRIVER` was `WINDOWS_DDNPM_VERSION`.